### ISSUE TITLE
Expanding scGPT Predictions to Additional Genes Beyond Tutorial Examples

### ISSUE THREAD

**Issue by KristinTsui** [bowang-lab/scGPT#176]
Thank you for the comprehensive tutorials on finetuning the scGPT model. I've been working through the [Tutorial_Perturbation.ipynb](https://github.com/bowang-lab/scGPT/blob/main/tutorials/Tutorial_Perturbation.ipynb) and have successfully fine-tuned the model using the Adamson dataset. My objective now is to leverage this fine-tuned model to explore the perturbation effects on a broader set of genes of interest, beyond those directly covered in the tutorial examples.

In block 19 of the tutorial, under the "Evaluations" section, the `plot_perturbation` function seems to require that the `query` parameter match annotations found within adata.obs for conditions and adata.uns for mappings of top non-dropout differentially expressed genes. 

Could you provide some guidance on how I could apply the model for prediction and visualization of perturbation effects on additional genes? More specifically, I would like to predict gene expression changes for a few gene present in adata.var.

Thank you in advance for any assistance or suggestions you can provide!
